routes/web.php

```diff
@@ -54,6 +55,17 @@
 
     Route::resource('trade', App\Http\Controllers\TradeController::class);
 
+    Route::put('/offer/update', [App\Http\Controllers\TradeOfferController::class, 'update'])
+        ->name("offer.update");
+
+    Route::get('/offer/edit', [App\Http\Controllers\TradeOfferController::class, 'edit'])
+        ->name("offer.edit");
+
+    Route::get('/offer/create', [App\Http\Controllers\TradeOfferController::class, 'create'])
+        ->name("offer.create");
+
+    Route::resource('offer', App\Http\Controllers\TradeOfferController::class)->except('update', 'edit', 'create');
```

Comment:
I did this because some of the default routes (before I added ->except(...)) expected parameters even though I didn't necessarily have any to pass. I left this because it works fine for store and I was thinking we might define more controller functions later on. I suppose I can just remove this and manually add a store route for the moment.